lib/api/middleware/set-linz-model.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const linz = require('../../../');
+
+/**
+ * Set req.linz.model
+ * @returns {Void} Calls the next middleware.
+ */
+const setLinzModel = () => (req, res, next) => {
```

Comment:
Why does this need to be a function that returns a function? There are no arguments, so it would seem unnecessary?